src/containers/IdentityContainer.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+
+import { esriPromise } from 'react-arcgis';
+
+import { Identity } from '../components/Identity';
+
+interface IdentityComponentProps {
+  credential?: __esri.Credential;
+  portalUrl: string;
+  handleLoginChange: (credential?: __esri.Credential) => void;
+}
+interface IdentityComponentState {}
+
+export class IdentityContainer extends React.Component<IdentityComponentProps, IdentityComponentState> {
+  identityManager: __esri.IdentityManager;
+
+  constructor(props: IdentityComponentProps) {
+    super(props);
+    this.handleLogin = this.handleLogin.bind(this);
+    this.handleLogout = this.handleLogout.bind(this);
+  }
+
+  componentDidMount() {
+    esriPromise([
+      'esri/identity/OAuthInfo',
+      'esri/identity/IdentityManager'
+    ]).then(([OAuthInfo, IdentityManager]) => {
+      this.identityManager = IdentityManager;
+
+      const info = new OAuthInfo({
+          appId: 'y4Lx1l6456Mbf85z',
```

Comment:
I think that if we've gone so far as to pass the `portalUrl` in as a `prop`, we may as well send this as a `prop` so as to keep all config in `App.tsx` (eventually I imagine we'll create a `config.tsx`).  What do you think?

Reply:
:white_check_mark: